From 256f5efbf23ff68c489dad92f99d1cecfb021729 Mon Sep 17 00:00:00 2001
From: Janne Grunau <j@jannau.net>
Date: Tue, 8 Feb 2022 21:23:15 +0100
Subject: [PATCH] fixup! arm64: dts: apple: t600x-j314-j316: Add NOR flash node

status = "ok" is not spec complient, disable the flash node to make it
harder to shoot oneself in the foot.

Signed-off-by: Janne Grunau <j@jannau.net>
---
 arch/arm64/boot/dts/apple/t600x-j314-j316.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/apple/t600x-j314-j316.dtsi b/arch/arm64/boot/dts/apple/t600x-j314-j316.dtsi
index 8452a93fd320ea..4ac3a1bebc150b 100644
--- a/arch/arm64/boot/dts/apple/t600x-j314-j316.dtsi
+++ b/arch/arm64/boot/dts/apple/t600x-j314-j316.dtsi
@@ -131,7 +131,7 @@
 };
 
 &spi1 {
-	status = "ok";
+	status = "disabled";
 
 	flash@0 {
 		compatible = "jedec,spi-nor";
